polkadot-protocol/smart-contracts-basics/index.md

```diff
@@ -0,0 +1,8 @@
+-
+title: Smart Contracts Basics
+description: TODO
```

Comment:
If this is for staging and will be updated before shipping then this is fine. Otherwise, please address.